rwengine/src/ai/CharacterControllerActivities.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#ifndef _CONTROLLER_ACTIVITY_HPP_
+#define _CONTROLLER_ACTIVITY_HPP_
+
+#include <ai/CharacterController.hpp>
+
+
+#define DECL_ACTIVITY(act_name)                                               \
+	class act_name : public CharacterController::Activity                 \
+	{                                                                     \
+	public:                                                               \
+		static constexpr auto ActivityName = #act_name;               \
+		std::string name() const override { return ActivityName; };   \
+		bool update(CharacterObject*, CharacterController*) override;
```

Comment:
This opens a curly brace but doesn't close it.
It will throw of some code-highlighters and it's very confusing while reading the code.
This is not going to get an okay from me.